src/components/network/network-loader.js

```diff
@@ -168,7 +170,7 @@ genotet.NetworkLoader.prototype.addGenes_ = function(genes) {
   });
 
   if (!newGenes.length) {
-    genotet.warning('genes are already in the network');
+    genotet.warning('genes are already in the network or not existing');
```

Comment:
I am sorry the situation that genes are not existing is already warned by another function. So for here, just warning about already in the network.